HIP-xxx - Pause SP Hex Boosting Rewards For All Radios.md

```diff
@@ -0,0 +1,81 @@
+# HIP-xxx - Pause HIP-86 SP Hex Boosting Rewards For All Radios
+
+- Author(s): [@mrfizzy99](https://github.com/mrfizzy99) (Fizzy99 - Fiz-Tech.net) , [@0lav](https://github.com/0lav) (SaintOlav), [@capjbadger007](https://github.com/capjbadger007) (ElonTusk) , [@bigbuffer](https://github.com/bigbuffer) (JD - Moken) , [Max Gold](https://github.com/GoldHawksAssociates) (Max - GoldHawksAssociates) , [@Gateholder](https://github.com/Gateholder) (Gateholder Worthington Networks)
+- Start Date: 2024/02/13
+- Category: Economic
+- Original HIP PR: <!-- leave this empty; maintainer will fill in ID of this pull request -->
+- Tracking Issue: <!-- leave this empty; maintainer will create a discussion issue -->
+- Vote Requirements: veMOBILE Holders
+
+## Summary
+
+This HIP aims to pause and remove Service Provider Hex Boosting that was passed in HIP-84, to be completely reworked with current available equipment that was not available previously in mind and new economics to better scale nationwide and beyond.
+
+## Prior Related HIPs
+
+HIP-84 created Service Provider Hex Boosting.
+
+
+## Motivation
+
+- As of 02/14/2024, when the Nova Dev team announced that HIP-84 implementation is now complete, it was not correctly done so and omitted the specified requirements of "at least three unique phones with discovery mapping enabled have successfully connected and passed at least 1MB of data at the location of coverage (as evidenced by the Mobile Oracle)."
```

Comment:
Unfortunately there is no good way in the HIP process for the core devs to define/justify multiple rollout steps. Other HIPs have been rolled out in more than one step as well, and since the particular requirement is (1) a somewhat ambiguous, (2) involves PII data in a new data flow to the oracles. 

So if rolling back a deployed HIP instead of stepping forward in phases is preferred I'd personally be disappointed in the inability to learn in each of the deployment steps, but Id' understand.  It'd be good to actually define how adjustments and clarifications to HIPs are made as part of the overall HIP and implementation process

Reply:
Unfortunately according announcement provided by Nova and/or Foundation comms, implementation is now complete.  With further mention that it is incomplete nor has more to be done. 

Lets be real here, Nova/Helium Mobile is the one that started putting Boosted and essentially promising deployers on the planner way before it was even ready.